tests/unit/lms/tasks/email_digests_test.py

```diff
@@ -0,0 +1,55 @@
+import logging
+from contextlib import contextmanager
+from datetime import datetime
+from unittest.mock import sentinel
+
+import pytest
+
+from lms.tasks.email_digests import send_instructor_email_digests
+
+
+class TestSendInstructorEmailDigests:
+    def test_it(self, caplog):
+        caplog.set_level(logging.INFO)
+        since = datetime(year=2023, month=3, day=1)
+        until = datetime(year=2023, month=3, day=2)
+
+        send_instructor_email_digests(
+            sentinel.h_userids,
+            since.isoformat(),
+            until.isoformat(),
+            sentinel.override_to_email,
+        )
+
+        assert caplog.record_tuples == [
+            (
+                "lms.tasks.email_digests",
+                logging.INFO,
+                "send_instructor_email_digests(sentinel.h_userids, datetime.datetime(2023, 3, 1, 0, 0), datetime.datetime(2023, 3, 2, 0, 0), override_to_email=sentinel.override_to_email)",
+            )
+        ]
```

Comment:
This assertion will be replaced by asserting that `DigestService` was called correctly